tests/unit/compile/test_utils.py

```diff
@@ -0,0 +1,12 @@
+import tempfile
+
+
+from flambe.compile.utils import write_deps
+
+
+def test_write_deps():
+    dummy_dependencies = ['numpy==1.2.3', 'pip~=1.1.1', 'some_other-random dep']
+    with tempfile.NamedTemporaryFile() as tmpfile:
+        write_deps(tmpfile.name, dummy_dependencies)
```

Comment:
i think you should test it without the input as well - preferably with a mock on freeze

Reply:
yes good idea.